src/openrct2/object/ImageTable.cpp

```diff
@@ -219,6 +230,47 @@ std::vector<std::unique_ptr<ImageTable::RequiredImage>> ImageTable::ParseImages(
     return result;
 }
 
+std::vector<std::unique_ptr<ImageTable::RequiredImage>> ImageTable::LoadImageArchiveImages(
+    IReadObjectContext* context, const std::string& path, const std::vector<int32_t>& range)
+{
+    std::vector<std::unique_ptr<RequiredImage>> result;
+    auto gxRaw = context->GetData(path);
+    std::optional<rct_gx> gxData = gfx_load_gx(gxRaw);
+    if (gxData.has_value())
+    {
+        size_t placeHoldersAdded = 0;
```

Comment:
After my PR this part of the function will change. This will use common code from `LoadObjectImages`